vividus/src/test/java/org/vividus/runner/BddScenariosCounterTests.java

```diff
@@ -0,0 +1,157 @@
+/*
+ * Copyright 2019 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.vividus.runner;
+
+import static org.hamcrest.CoreMatchers.containsString;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.powermock.api.mockito.PowerMockito.when;
+import static org.powermock.api.mockito.PowerMockito.whenNew;
+
+import java.util.Properties;
+
+import com.github.valfirst.jbehave.junit.monitoring.JUnitReportingRunner;
+
+import org.apache.commons.cli.UnrecognizedOptionException;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.Description;
+import org.junit.runner.RunWith;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PowerMockIgnore;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+import org.vividus.SystemOutTests;
+import org.vividus.configuration.BeanFactory;
+import org.vividus.configuration.Vividus;
+
+@RunWith(PowerMockRunner.class)
+@PrepareForTest({ Vividus.class, BeanFactory.class, BddScenariosCounter.class })
+@PowerMockIgnore({ "javax.*", "com.sun.*", "jdk.*", "org.xml.*", "org.w3c.*" })
+public class BddScenariosCounterTests extends SystemOutTests
+{
+    private static final String DEFAULT_STORY_LOCATION = "story";
+    private static final String DIR_VALUE = "story/bvt";
+    private static final String SEPARATOR = " | ";
+    private static final String STORIES = "Stories";
+    private static final String SCENARIOS = "Scenarios";
+    private static final String SCENARIOS_WITH_EXAMPLES = "Scenarios with Examples";
+    private static final String PROPERTIES = "properties";
+    private static final String ROOT = "root description";
+    private static final String STORY = "story description";
+    private static final String SCENARIO = "scenario description";
+    private static final String EXAMPLE = "example description";
+    private static final String BEFORE = "before description";
+    private static final String RESOURCE_LOCATION = "bdd.story-loader.batch1.resource-location";
+
+    @Before
+    public void before()
+    {
+        PowerMockito.mockStatic(Vividus.class);
```

Comment:
is it required?

Reply:
Without it, I get this exception
`java.lang.IllegalStateException: environments is not set`